packages/thunderstore-api/src/apiFetch.ts

```diff
@@ -16,15 +16,18 @@ export async function apiFetch2(args: apiFetchArgs) {
   const url = getUrl(args.config, args.path, args.query);
 
   const response = await fetch(url, {
+    method: args.method ? args.method : "GET",
     headers: {
       ...BASE_HEADERS,
       ...getAuthHeaders(args.config),
     },
-    body: args.body,
+    ...(args.method === "GET" || args.method === "HEAD" || args.method === null
+      ? {}
+      : { body: args.body }),
```

Comment:
This is wrong, whether a request body should be sent has nothing to do with the request method. It's just more common for that to be done with POST requests.

If having `undefined` in the body was causing issues, check if `null` works instead:

```suggestion
    body: body ?? null,
```

According to the typescript type hints at least, that should be valid.

Reply:
This suggestion isn't necessary if you follow the larger change suggestion, leaving it as a note anyway.